gcp-jobs/filings-notebook-report/src/notebookreport/notebookreport.py

```diff
@@ -0,0 +1,190 @@
+"""The Notebook Report - This module is the API for the Filings Notebook Report."""
+
+import ast
+import fnmatch
+import logging
+import os
+import smtplib
+import sys
+import time
+import traceback
+import warnings
+from datetime import datetime, timedelta
+from email import encoders
+from email.mime.base import MIMEBase
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+import papermill as pm
+from dateutil.relativedelta import relativedelta
+from flask import Flask, current_app
+
+from .config import Config
+from .util.logging import setup_logging
+
+setup_logging(os.path.join(os.path.abspath(os.path.dirname(__file__)), 'logging.conf'))  # important to do this first
+
+# Notebook Scheduler
+# ---------------------------------------
+# This script helps with the automated processing of Jupyter Notebooks via
+# papermill (https://github.com/nteract/papermill/)
+
+
+def create_app(config=Config):
+    """Create app."""
+    app = Flask(__name__)
+    app.config.from_object(config)
+    # db.init_app(app)
```

Comment:
Can be removed?

**Not in scope**, but ideally it would import the model, init, and this would be used in the notebooks instead of raw sql I think

Reply:
Removed, yes I was trying to see if there was a convenient way to get off of the %%sql which requires DEPRECATED_DEFAULT enabled